tests/thread/thread_locktimed1.py

```diff
@@ -0,0 +1,40 @@
+# test timed _thread lock object using a single thread
+#
+# MIT license; Copyright (c) 2016 Damien P. George on behalf of Pycom Ltd
```

Comment:
This copyright line can be removed.  There's nothing of substance in this test copied from the other thread tests.